code/game/gamemodes/miniantags/guardian/types/protector.dm

```diff
@@ -62,14 +62,18 @@
 		toggle = FALSE
 		QDEL_LIST_CONTENTS(connected_shields)
 	else
+		if(!isturf(loc))
+			return
+		if(get_turf(summoner) == get_turf(src))
+			return
```

Comment:
Can you add a to_chat here to indicate why it's not doing anything?

Reply:
done